components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -1666,35 +1685,37 @@ this mask will have smb calculated over the entire global land surface
 <!-- Created by lnd/clm/bld/namelist_files/createMapEntry.pl--> 
 
 <map frm_hgrid="0.5x0.5"    frm_lmask="AVHRR"  to_hgrid="0.125x0.125"   to_lmask="nomask" 
->lnd/clm2/mappingdata/maps/0.125x0.125/map_0.5x0.5_AVHRR_to_0.125x0.125_nomask_aave_da_c140702.nc</map>
+>lnd/clm2/mappingdata/maps/0.125x0.125/map_0.5x0.5_AVHRR_to_0.125x0.125_nomask_aave_da_c190725.nc</map>
```

Comment:
Is this switching to a netcdf3 version of this file?

Reply:
What ncdump option can I use to verify if the file is netcdf3?

Reply:
I use `ncdump -k`

Reply:
Found it. `ncdump -k`

Reply:
The file is in `classic` format, but the file is not used by the model. It is used during preprocessing to create land surface dataset. Do you want me to regenerate all files?

Reply:
No only if its read in by the model.   But this raises a new question:  why are files not needed by the model in the namelist_defaults_clm4_5.xml?

Reply:
The `mksurfdata_map` (pre-processing tools) needs information about mapping files for remapping raw datasets onto new land surface resolution. The information about the mapping files in stored in the `namelist_defaults_clm4_5.xml`.

Reply:
To answer @rljacob 's question, my understanding is that putting these mapping file info in the namelist_defaults_clm4_5.xml ensures that the pre-processing tools and the model are consistent by using the same *.xml files to provide settings.